.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: false
```

Comment:
By using `sudo` we are no longer in the [container based infrastructure][1], which runs much much faster. We switched over in 3c0849b and it made a huge difference in how fast the tests were completed. I would strongly recommend staying with the container based infrastructure. [Here's how][2].

[1]: https://docs.travis-ci.com/user/workers/container-based-infrastructure
[2]: https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-on-Container-Based-Infrastructure